app/scripts/controllers/pod.js

```diff
@@ -204,54 +205,51 @@ angular.module('openshiftConsole')
       });
     };
 
+    var podResolved = function(pod, action) {
+      $scope.loaded = true;
+      $scope.pod = pod;
+      setLogVars(pod);
+      setContainerVars();
+      if (action === "DELETED") {
+        $scope.alerts["deleted"] = {
+          type: "warning",
+          message: "This pod has been deleted."
+        };
+      }
+      var pods = {};
+      pods[pod.metadata.name] = pod;
+      return pods;
```

Comment:
this is a little odd to me. there isnt really a reason to have this here when the object stuff is only being used for the image stream resolve